### ISSUE TITLE
Add a Click event to the cartesianChart itself

### ISSUE THREAD

**Issue by runoono** [Live-Charts/Live-Charts#822]
I cant find a built in function for clicking an empty point on the chart. Ive made a workaround by using the event from CartesianChart.Base.MouseButtonLeftUp then taking coordinates from Cursor.Position, running that through pointToClient and calculating where it would roughly fall by using doing stuff with how long is chart and distance between the minValue and maxValue on the axises. Looks something like this

`//converts Cursor.Postion to points on the chart       
        private double convertToChartX(Point P)
        {
            P = ohlcChart.PointToClient(P);
            return (P.X * (ohlcChart.AxisX[0].ActualMaxValue - ohlcChart.AxisX[0].ActualMinValue) / ohlcChart.Width) + ohlcChart.AxisX[0].ActualMinValue;
        }
        //converts Cursor.Postion to points on the chart
        private double convertToChartY(Point P)
        {
            P = ohlcChart.PointToClient(P);
            return ((ohlcChart.Height - P.Y) * (ohlcChart.AxisY[0].ActualMaxValue - ohlcChart.AxisY[0].ActualMinValue) / ohlcChart.Height)+ ohlcChart.AxisY[0].ActualMinValue;
        }`
now youre thinking if this works so well what's the problem? the problem is that this doesnt compensate for the blank spots made by the legend on the axises or the blank space under and on top of the charts
![image](https://user-images.githubusercontent.com/16387918/42285485-7234182e-7fb8-11e8-86ac-4a57a6e1b1d4.png)
what happens is the point Im clicking doesnt align with where the point is being places. In the picture Im point to the corner marked here and the point is to the lower right instead. I'd very much appreciate a lot if it was possible add click event that compensated for that.

**Comment by AlexandrVLopatin** [comment]
any news?

**Comment by runoono** [comment]
not really, don't even work on it anymore. Back when I was working on this I was pitching it as a replacement for an in house framework and I just couldn't get it to work as well as I wanted it to so I had to drop the project
